pkg/validation/validate_cluster.go

```diff
@@ -133,9 +126,7 @@ func (v *clusterValidatorImpl) Validate() (*ValidationCluster, error) {
 
 	// Do not use if we are running gossip
 	if !dns.IsGossipHostname(clusterName) {
```

Comment:
Not sure if we should use `host` instead of `clusterName` here ?

Reply:
If we are using gossip, won't `host` be the load balancer DNS?

Reply:
You’re probably right, I don’t much about gossip mode.